### ISSUE TITLE
Title is not converted to slug correctly for some characters

### ISSUE THREAD

**Issue by serapio** [tainacan/tainacan#762]
Document titles with characters outside the Latin1 range are mapped to URL paths that can't be found

What is happening?

1. In a new collection or in an existing one, create a document.
2. Provide a title that has non-Latin1 characters, e.g. "jʉꞌʉ" (note that is the saltillo char, not a single quote) or "故事"
3. Save changes
4. Click on the link for the document. This link has a slug that does not mutate these characters.
5. You get an Error 404 page.

**Expected behavior**

It should provide the summary page for the document. 

**Desktop:**

 - OS: Windows 11
 - Browser: Firefox
 - Version: 108

**Server**

 - Tainacan 0.19.3
 - WordPress 6.1.1

**Comment by mateuswetah** [comment]
Hey @serapio thank you for reporting this bug!

We'll take a look as soon as we can and give you some feedback.

**Comment by mateuswetah** [comment]
Hi @serapio!

I'm sorry for taking so long to look into this. We're finally at a moment of reorganizing our issues in the project and I would like to take a look at this. My first test, using the characters that you provided, did not result in any issue, I was able to click the Item page link and view it on the theme side. 

I've seen similar issues in a situation that ended up being related to some COLLATE settings in the database. In any case, can you please try to reproduce it again in the latest version of the plugin?

**Comment by serapio** [comment]
Hi @mateuswetah , I'm still getting that behavior, even on a fresh install of tainacan.
It's currently public here: https://contando.me/index.php/prueba/
The link is https://contando.me/index.php/prueba/n%ca%89%ea%9e%8c%ca%89-%e6%95%85%e4%ba%8b/ but that's a 404

**Comment by mateuswetah** [comment]
@mateuswetah ok, let's see.... in your wp-config.php, would have this set?

“define( ‘DB_CHARSET’, ‘utf8’ );”

If not please add it and try creating the item again.

**Comment by serapio** [comment]
Yeah, that's already set. It looks to me like it's just a problem with the post name as a slug in the url then not matching in the db, as the text is preserved elsewhere.
![imagen](https://github.com/user-attachments/assets/48c05050-acd5-4f1e-8277-54111dfee682)

**Comment by mateuswetah** [comment]
Since you have access to phpMyAdmin, can you check if it is using `utf8mb4_unicode_ci`?
https://www.wesbytes.com/guide/kb/how-to-change-the-mysql-collation-settings-in-phpmyadmin/

**Comment by serapio** [comment]
Sorry for the delay following up. I figured out what it was, and I don't know how it broke, but it looks like something that shouldn't affect people more generally. I had permalinks with a "personalized structure" of `/index.php/%year%/%monthnum%/%day%/%postname%/`, which apparently forwards correctly when postname is ascii, but not for utf8. (And yes, the collation is `utf8mb4_unicode_ci`.) When `index.php` is removed from the path it works, and I don't remember why it was there to begin with.

**Comment by mateuswetah** [comment]
Uhmm... ok then. It is a matter of how the permalinks are set. Usually we recommend using the "postname" setting but I've seen some users playing around with this options.

Would you mind closing the issue?
